### PR TITLE
Do not ignore empty ingress class

### PR DESCRIPTION
**Pull request by dintel**
This PR solves #291

### CODE REVIEW

**Comment by mikkeloscar** [comment]
Thanks for the PR. would you mind adding a test case covering this?

**Comment by dintel** [comment]
@mikkeloscar Done, please review.

File: README.md [hunk 1]
```diff
@@ -420,6 +420,7 @@ This value will be used to tag those resources.
 If you don't pass an ID, the default `kube-ingress-aws-controller` will be used.
 
 Usually you would want to combine this flag with `ingress-class-filter` so different types of ingresses are associated with the different controllers.
+To make `kube-ingress-aws-controller` manage both specific ingress class and an empty one (or ingresses without ingrss class annotation) add an empty class to the list. For example to manage ingress class `foo` and ingresses without class set parameter like this `--ingress-class-filter=foo,` (notice the comma in the end).
```
Comment (mikkeloscar) [inline README.md]: nit: typo: `ingrss`

Comment (dintel) [reply README.md]: @mikkeloscar fixed.

File: README.md [hunk 2]
```diff
@@ -420,6 +420,7 @@ This value will be used to tag those resources.
 If you don't pass an ID, the default `kube-ingress-aws-controller` will be used.
 
 Usually you would want to combine this flag with `ingress-class-filter` so different types of ingresses are associated with the different controllers.
+To make `kube-ingress-aws-controller` manage both specific ingress class and an empty one (or ingresses without ingrsss class annotation) add an empty class to the list. For example to manage ingress class `foo` and ingresses without class set parameter like this `--ingress-class-filter=foo,` (notice the comma in the end).
```
Comment (szuecs) [inline README.md]: ingrsss -> ingress

File: kubernetes/adapter_test.go [hunk 3]
```diff
@@ -306,3 +306,77 @@ func TestAdapter_GetConfigMap(t *testing.T) {
...
+			ingressClassFilters: []string{"other", ""},
+			expectedIngressNames: []string{
+				"fixture01",
+				"fixture03",
+			},
+		},
+	} {
+		t.Run(name, func(t *testing.T) {
+			// test := test
+			// t.Parallel()
```
Comment (szuecs) [inline kubernetes/adapter_test.go]: please drop commented code

**Comment by szuecs** [comment]
Only nits, but please fix them, thanks!

File: kubernetes/adapter_test.go (hunk 3 above)
Comment (dintel) [reply kubernetes/adapter_test.go]: Done.

File: README.md (hunk 2 above)
Comment (dintel) [reply README.md]: Done.

**Comment by dintel** [comment]
Also reworded commit messages to reference correct issue number.

File: README.md (hunk 2 above)
Comment (mikkeloscar) [reply README.md]: It still says "ingrsss"

Comment (dintel) [reply README.md]: Nailed it now.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**